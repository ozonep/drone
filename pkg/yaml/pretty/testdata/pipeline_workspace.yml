kind: pipeline
name: default

workspace:
  base: /go
  path: src/github.com/octocat/hello-world

steps:
- name: test
  image: golang
  commands:
    - go build
    - go test -v
