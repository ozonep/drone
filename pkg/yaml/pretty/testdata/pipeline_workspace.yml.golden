---
kind: pipeline
name: default

platform:
  os: linux
  arch: amd64

workspace:
  base: /go
  path: src/github.com/octocat/hello-world

steps:
- name: test
  image: golang
  commands:
  - go build
  - go test -v

...
